scripts/xtensa-build-zephyr.py

```diff
@@ -585,8 +582,32 @@ def build_rimage():
 			rimage_build_cmd.append("-v")
 	execute_command(rimage_build_cmd, cwd=west_top)
 
-
-def rimage_options(platform_dict):
+def concat_rimage_configs(platform_name, dest_dir):
```

Comment:
nak, lets use the pre-processor. We also need to add Kconfig detail into the toml too.